development/image-builder/main.go

```diff
@@ -444,6 +456,7 @@ func (o *options) gatherOptions(fs *flag.FlagSet) *flag.FlagSet {
 	fs.StringVar(&o.logDir, "log-dir", "/logs/artifacts", "Path to logs directory where GCB logs will be stored")
 	fs.StringVar(&o.orgRepo, "repo", "", "Load repository-specific configuration, for example, signing configuration")
 	fs.Var(&o.tags, "tag", "Additional tag that the image will be tagged with. Optionally you can pass the name in the format name=value which will be used by export-tags")
+	fs.Var(&o.buildArgs, "build-arg", "Flag to pass additional arguments to built Dockerfile. Van be used in the name=value format.")
```

Comment:
```suggestion
	fs.Var(&o.buildArgs, "build-arg", "Flag to pass additional arguments to built Dockerfile. It can be used in the name=value format.")
```